webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/reports/ReportingServiceImpl.scala

```diff
@@ -262,7 +265,7 @@ trait CachedFindRuleNodeStatusReports extends ReportingService with CachedReposi
    * It's a List and not a Set, because we want to keep the precedence in
    * invalidation request.
    */
-  private[this] val invalidateComplianceRequest = Queue.dropping[List[NodeId]](1).runNow
+  private[this] val invalidateComplianceRequest = Queue.dropping[List[(NodeId, Option[NodeStatusReport])]](1).runNow
```

Comment:
Here i'd like an action that can be
NewRunComputed[NodeStatusReports]
InvalidateCompliance[Option[NodeConfiguration]]
NoReports